e2e/container/create_test.go

```diff
@@ -0,0 +1,96 @@
+package container
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/docker/cli/e2e/internal/fixtures"
+	"github.com/gotestyourself/gotestyourself/icmd"
+)
+
+func TestCreateWithContentTrust(t *testing.T) {
```

Comment:
If both `create` and `run` use exactly the same code, do we need e2e tests for both?